tests/secrets/custom_and_entropy/main.tf

```
@@ -0,0 +1,3 @@
+resource "aws_instance" "a" {
+  test_pass: "z2b7k2cQfzc+yjP2K8cjuQ8uoorHBpEvC+XWhU3Z5+IdrPQYwr991Lj73xfZ+RA2GzC0wTedDTvb1C2NX+3Gpw==",
```

```suggestion
  test_pass = "z2b7k2cQfzc+yjP2K8cjuQ8uoorHBpEvC+XWhU3Z5+IdrPQYwr991Lj73xfZ+RA2GzC0wTedDTvb1C2NX+3Gpw=="
```
this should handle the TF parsing error